recipe/build-core.bat

```diff
@@ -14,16 +14,12 @@ echo dir %BAZEL_VC%
 dir "%BAZEL_VC%"
 
 echo ==========================================================
-echo calling bash to build
+echo calling pip to install
```

Comment:
is this change to pip build still required? would this all work now with the original build script since patch 0006?

Reply:
The original build script used pip. I [changed it to bash](https://github.com/conda-forge/ray-packages-feedstock/pull/135/files#diff-0f49d6f77e617c54088340290b14937fdfae45eaed495ef2a679995df621b13fR16) in #135 when trying to figure out what was going on with windows.